src/Common/Exceptions/BaseException.h

```diff
@@ -0,0 +1,19 @@
+#ifndef __BASE_EXCEPTION_H_
+#define __BASE_EXCEPTION_H_
+
+#include <iostream>
+
+namespace FileManager {
+
+    class BaseException {
```

Comment:
Shouldn't this be inheriting from the system exception class?

Reply:
Yeah sorry 😀